src/main/java/marquez/api/mappers/NamespaceMapper.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package marquez.api.mappers;
+
+import static java.util.Objects.requireNonNull;
+
+import marquez.api.models.NamespaceRequest;
+import marquez.api.models.NamespaceResponse;
+import marquez.service.models.Namespace;
+
+public final class NamespaceMapper extends Mapper<NamespaceResponse, Namespace> {
+  private NamespaceMapper() {}
+
+  public static Namespace map(NamespaceResponse namespace) {
+    requireNonNull(namespace, "namespace must not be null");
+    return new Namespace(
+        null, namespace.getName().toLowerCase(), namespace.getOwner(), namespace.getDescription());
+  }
+
+  public Namespace of(String namespaceName, NamespaceRequest request) {
```

Comment:
Nice! So, since you copied over the code from `NamespaceApiMapper`. Let's make sure we update and name all converter methods as `map()`.

Here's what I'd recommend for your `map()` method:
-  Use the type [`NamespaceName`](https://github.com/MarquezProject/marquez/blob/master/src/main/java/marquez/common/models/NamespaceName.java) instead of a string
- Annotate all method args with [`@NonNull`](https://projectlombok.org/features/NonNull)

Reply:
Ok, so gathering what you said and referring to `DbTableVersionMapper.java`, I made the following changes:
1. Removed the `extends Mapper<>`
2. Added `@NonNull` before all method args
3. changed `public Namespace of()` to `public static Namespace map()`
4. Commented out the first method

```java
public final class NamespaceMapper {
  private NamespaceMapper() {}

/*  public static Namespace map(NamespaceResponse namespace) {
    requireNonNull(namespace, "namespace must not be null");
    return new Namespace(
        null, namespace.getName().toLowerCase(), namespace.getOwner(), namespace.getDescription());
  }*/

  public static Namespace map(@NonNull NamespaceName namespaceName, @NonNull NamespaceRequest request) {
    return map(
        new NamespaceResponse(
            namespaceName, null, request.getOwner(), request.getDescription().orElse(null)));
  }
}
```

Reply:
You're making great progress, @DavidLi119!  I noticed the CI build is failing. Are you able to run the following cmd successfully locally on your machine?

```
$ ./gradlew compileJava
```